Implemented "Generate SELECT function(...)" data grid action

tuffnatty opened the pull request:
It's a feature allowing to quickly evaluate the result of function application to selected data grid cells. E.g.:

![Screenshot 2024-08-19 at 16 57 15](https://github.com/user-attachments/assets/e41b0bda-6a9f-40d8-a1a5-e846ff611896)

Probably the interface still needs improvement, but as it is, it's already a great usability improvement.

pawelsalawa commented:
The only small thing I would change for now is to remove the icon from it. It messes up the alignment of the menu. Other than that - cool!

tuffnatty commented:
I have removed the icon. I have also modified the generated query so that the function is evaluated at most once per selected item. The functions offered in the menu are functions accepting at least one argument.
